tidb-lightning/tidb-lightning-command-line-full.md

```diff
@@ -0,0 +1,60 @@
+---
+title: TiDB Lightning Command Line Flags
+summary: Learn how to configure TiDB Lightning using command line flags.
+---
+
+# TiDB Lightning Command Line Flags
+
+You can configure TiDB Lightning either using the configuration file or in command line. This document describes the command line flags of TiDB Lightning.
+
+## Command line flags
+
+### `tidb-lightning`
+
+You can configure the following parameters using `tidb-lightning`:
+
+| Parameter | Description | Corresponding configuration item |
+| :---- | :---- | :---- |
+| `--config <file>` | Read the global configuration from the file. If this parameter is not specified, TiDB Lightning uses the default configuration. | |
+| `-V` | Print the program version. | |
+| `-d <directory>` | The local directory or [external storage URL](/br/backup-and-restore-storages.md) of data files. | `mydumper.data-source-dir` |
+| `-L <level>` | The log level: `debug`, `info`, `warn`, `error`, or `fatal`. `info` by default.| `lightning.level` |
+| `-f <rule>` | [Table filter rules](/table-filter.md). Can be specified multiple times. | `mydumper.filter` |
+| `--backend <backend>` | Select an import mode. `local` refers to [physical import mode](/tidb-lightning/tidb-lightning-physical-import-mode.md); `tidb` refers to [logical import mode](/tidb-lightning/tidb-lightning-logical-import-mode.md). | `tikv-importer.backend` |
```

Comment:
```suggestion
| `--backend <backend>` | The import mode. `local` refers to [physical import mode](/tidb-lightning/tidb-lightning-physical-import-mode.md); `tidb` refers to [logical import mode](/tidb-lightning/tidb-lightning-logical-import-mode.md). | `local` |
```

Reply:
The correct value is `tikv-importer.backend`. The Chinese version is fixed in https://github.com/pingcap/docs-cn/pull/11150/files